auv_teleop/scripts/joy_manager.py

```diff
@@ -85,9 +85,9 @@ def joy_callback(self, msg):
         with self.lock:
             self.joy_data = msg
 
-            self.torpedo1_button_event.update(self.joy_data.buttons[0])
-            self.torpedo2_button_event.update(self.joy_data.buttons[1])
-            self.droper_button_event.update(self.joy_data.buttons[3])
+            self.torpedo1_button_event.update(self.joy_data.buttons[4])
```

Comment:
Looks good, but I'm going to be annoying for the next 5 minutes :D 

why not make these ros param? for example


```yaml
buttons:
  launch_torpedo1: 5
  launch_torpedo2: 3
  drop_ball: 2


axes:
  surge:  # (forward)
    index: 1 # button index
    gain: 0.1 # scalar
  sway: # lateral
    index: 2
    gain : 0.5
  yaw:
    index: 3 
    gain: 0.7


```